### PR TITLE
Made more PEP-compliant and improved f-strings

### PR DESCRIPTION
**Pull request by ArztKlein**

### CODE REVIEW

**Comment by ArztKlein** [comment]
I forgot to mention that I also fixed the today command

File: sakura/utils/database.py [hunk 1]
```diff
@@ -36,13 +36,14 @@ async def after_hook(cog, ctx):
 
 @before_invoke_hook(priority=1000)
 async def attach_database_user(cog, ctx):
-    """Before-hook to attach message author to ctx.db_user"""
+    """Before-hook to attach message author to ctx.db_use.r"""
```
Comment (HexF) [inline sakura/utils/database.py:39]: why is there `.r`

File: sakura/utils/secrets.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-"""Secret manager
```
Comment (HexF) [inline sakura/utils/secrets.py:1]: PEP 257 says this is normal

File: sakura/utils/hooks.py [hunk 3]
```diff
@@ -8,15 +8,16 @@
...
 
 async def invoke_many(functions, *args):
-    """Useful in before/after hooks for running multiple"""
+    """Useful in before/after hooks for running multiple."""
     for f in functions:
         await f(*args)
 
 
 def multi_hook(func):
-    """Invokes all previously registered hooks by other utilities
```
Comment (HexF) [inline sakura/utils/hooks.py:17]: PEP 257 says this is normal
leave start on `"""` line

File: sakura/utils/database.py (hunk 1 above)
Comment (HexF) [reply sakura/utils/database.py:39]: seems like the `.` is in the wrong place

File: sakura/cogs/glance.py [hunk 4]
```diff
@@ -78,7 +81,7 @@ async def weather(self, ctx, location=None):
     async def moon(self, ctx):
         """ Returns the current moon, as an emoji. """
         
-        t = datetime.datetime.now(pytz.timezone("utc"))
+        t = datetime.datetime.utcnow()
```
Comment (HexF) [inline sakura/cogs/glance.py:84]: remove pytz import if its not used

File: sakura/cogs/interactions.py [hunk 5]
```diff
@@ -17,6 +18,7 @@ async def interact(ctx, member, action, desc, find, self):
         embedVar.set_image(url=str(t.random("anime alone")))
         await ctx.send(embed=embedVar)
 
+
 class MemberInteractions(commands.Cog, name="Interactions"):
     """
```
Comment (HexF) [inline sakura/cogs/interactions.py:23]: PEP 257 says this is normal
leave start on """ line

File: sakura/cogs/misc.py [hunk 6]
```diff
@@ -8,8 +8,10 @@
 
 class Misc(commands.Cog, name="Miscellaneous"):
     """
```
Comment (HexF) [inline sakura/cogs/misc.py:10]: PEP257

File: sakura/cogs/misc.py [hunk 7]
```diff
@@ -8,8 +8,10 @@
...
-    Random Stuff I can't categorise
+    Comamnds that do not fit into their own categories.
```
Comment (HexF) [inline sakura/cogs/misc.py:11]: spelling

File: sakura/models/guild.py [hunk 8]
```diff
@@ -15,8 +14,6 @@ class Guild(Base):
 
     custom_prefix = Column(String, nullable=True)
 
-    
-
 
     def __repr__(self) -> str:
         return f"<Guild(id={self.id}, discord_id={self.discord_id}, custom_prefix={self.custom_prefix}>"
```
Comment (HexF) [inline sakura/models/guild.py:19]: `f"{=}"` format?

**Review by HexF — CHANGES_REQUESTED** [review]

File: sakura/cogs/misc.py (hunk 7 above)
Comment (ArztKlein) [reply sakura/cogs/misc.py:11]: There is no spelling error?

Comment (HexF) [reply sakura/cogs/misc.py:11]: > Comamnds

File: sakura/cogs/glance.py (hunk 4 above)
Comment (ArztKlein) [reply sakura/cogs/glance.py:84]: pytz is infact used in another method

**Pull request merged**